src/decide.ts

```diff
@@ -475,28 +480,43 @@ async function assessAndSaveResults(
 	return assessment;
 }
 
+export async function getGuidInfoHashMap(): Promise<Map<string, string>> {
+	return new Map(
+		(
+			await db("decision")
+				.select("guid", "info_hash")
+				.whereNotNull("info_hash")
+		).map(({ guid, info_hash }) => [guid, info_hash]),
+	);
+}
+
 /**
  * Some trackers have alt titles which get their own guid but resolve to same torrent
  * @param guid The guid of the candidate
+ * @param guidInfoHashMap The map of guids to info hashes
  * @returns The info hash of the torrent if found
  */
-async function fuzzyGuidLookup(guid: string): Promise<string | undefined> {
+async function fuzzyGuidLookup(
+	guid: string,
+	guidInfoHashMap: Map<string, string>,
+): Promise<string | undefined> {
 	if (!guid.includes(".tv/torrent/")) return;
-	const torrentIdStr = guid.match(/\.tv\/torrent\/(\d+)\/group/)?.[1];
+	const torrentIdRegex = /\.tv\/torrent\/(\d+)\/group/;
```

Comment:
where's the alt-title-unique part? after `/group/`?

Reply:
`.tv/torrent/132469/group?nh=8CA2A342113D4GFJOWKP911A4044F`

It seems to be a hash of something, .torrent files are the same.